deployment/molecule/default/tests/test_oracle.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('oracle')
+
+
+@pytest.mark.parametrize("name", [
+    ("oracle_rabbit_1"),
+    ("oracle_redis_1"),
+    ("oracle_bridge_request_1"),
+    ("oracle_bridge_collected_1"),
+    ("oracle_bridge_affirmation_1"),
+    ("oracle_bridge_senderhome_1"),
+    ("oracle_bridge_senderforeign_1"),
+])
+def test_docker_containers(host, name):
+    container = host.docker(name)
+    assert container.is_running
+
+
+@pytest.mark.parametrize("service", [
+    ("poabridge"),
```

Comment:
I am not sure that I understand which configuration (bridge contracts) will be used to for deployment testing. As part of testing we need to make sure that the tokenbridge oracle configured properly to work with the particular bridge contracts.

Reply:
Good question, I realized now that the tests are not ideal because:

The playbooks clone the monorepo on target-hosts using the `master` branch.

So on `branch-x`, if we modify playbooks, change oracle/ui/monitor code, update `contracts` submodule, then we test:
- Modified playbooks
- Unchanged oracle/ui/monitor from `master` branch
- Not-updated `contracts` submodule like in `master` branch

---

The desired behaviour, I presume, is that we test everything from `branch-x`.
I will try to make it happen in #119

Reply:
Good catch! My questions was actually whether the deployment e2e test works with contracts from `common-e2e`. The question appear since I do not see any configuration file for Ansible introduced by this PR allowing to work deployed contracts.

Reply:
I see. Currently the deployment tests do not use deployed contracts at all - just asserting that the playbook works and that the Oracle starts. It doesn't have E2E tests asserting that the Oracle works correctly (because we have that in `oracle-e2e`)

Seems like it could be beneficial to combine the molecule tests with e2e tests into ultimate e2e tests:
- Deploy the contracts using `e2e-commons`
- Deploy Oracle/UI/Monitor using Molecule, using the above contracts
- Run `oracle-e2e`, `ui-e2e` and `monitor-e2e` testing the above

But it doesn't seem quick nor easy.

Reply:
Exactly! Otherwise we are not testing the monorepo as the whole project by e2e tests. Instead the tests considers every component independently that does not prove that they will work together.

Reply:
I created #121 for this.

Reply:
thanks! 
Currently my main worry is that you just run the poabridge service but since it was connected to any real contract we don't know if Ansible prepares proper configuration for the service. Do you think it should be covered in a separate issue as well?

Reply:
Yes, I would prefer to do it in a separate issue/PR, much like we did with #12 and #115 - keeping PRs rather small and short-lived

Reply:
OK.